website/docs/help-and-support/troubleshooting-guide/action-errors/mongodb-errors.md

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
@jnikhila Instead of putting this in the troubleshooting guide, do you think this should be a dedicated upgrade page (like the MongoDB upgrade page)?

Reply:
Are we planning to add more details to the documentation for Upgrade that will be specific to Appsmith? If that's the case, then we can add it as a dedicated MongoDB Upgrade page.